src/group_theory/submonoid/operations.lean

```diff
@@ -735,6 +735,29 @@ lemma mker_prod_map {M' : Type*} {N' : Type*} [mul_one_class M'] [mul_one_class
   (g : M' →* N') : (prod_map f g).mker = f.mker.prod g.mker :=
 by rw [←comap_bot', ←comap_bot', ←comap_bot', ←prod_map_comap_prod', bot_prod_bot]
 
+/-- The `mul_hom` from the comap of a submonoid to itself. -/
+@[to_additive "the `add_hom` from the comap of an additive submonoid to itself.", simps]
```

Comment:
This is a lie:
```suggestion
/-- The `monoid_hom` from the comap of a submonoid to itself. -/
@[to_additive "the `add_monoid_hom` from the comap of an additive submonoid to itself.", simps]
```
`mul_hom` already exists but means something else. Same in the other docstrings